server/cmwell-controller/src/main/scala/cmwell/ctrl/checkers/ElasticsearchChecker.scala

```diff
@@ -29,12 +30,26 @@ import scala.concurrent.ExecutionContext.Implicits.global
   */
 object ElasticsearchChecker extends Checker with LazyLogging {
   override val storedStates: Int = 10
+
+  def getHealth (port: Int):Future[SimpleResponse[Array[Byte]]] = {
+
+    val url = s"http://${Config.pingIp}:$port/_cluster/health"
+    val esRes = Http.get(url)
+
+    esRes.flatMap{res =>
+      if ((res.status==200) || (port > 9203))
+        esRes
+      else
+        getHealth(port + 1)
```

Comment:
add log print of:
1. bad response code
2. the body of the response

Reply:
also not handled are the cases:
1. failed future
2. exception thrown by the Http library